src/splitter.ts

```diff
@@ -234,9 +242,12 @@ export const initializeSplitting = async (
 
   const splitVehiclesAndSend = (
     gtfsrtPulsarMessage: Pulsar.Message,
-    sendCallback: (fullApcMessage: Pulsar.ProducerMessage) => void,
+    sendCallback: (
+      fullApcMessage: Pulsar.ProducerMessage
```

Comment:
In our other services, apcMessage or fullApcMessage refers to messages with APC data. vehicle-position-splitter does not handle such messages at all. splitter handles the vehicle catalogue / vehicle-equipment mapping / vehicle-APC mapping / vehicle registry snapshot which has the expression "PASSENGER_COUNTER" in it. That's the only connection to automatic passenger counting context.

Please grep all occurrences of apc and check that you are not referring to apc messages. For example, in this case what is being sent is a GTFS RT VP message with a single vehicle inside.